federation/executor_test.go

```diff
@@ -166,11 +198,22 @@ func createExecutorWithFederatedUser() (*Executor, *schemabuilder.Schema, *schem
 	userWithContactInfo := s2.Object("User", UserWithContactInfo{}, schemabuilder.FetchObjectFromKeys(func(args struct{ Keys []*UserWithContactInfo }) []*UserWithContactInfo {
 		return args.Keys
 	}))
+	s2.Object("UserSkills", Skills{}, schemabuilder.FetchObjectFromKeys(func(args struct{ Keys []*Skills }) []*Skills {
+		return args.Keys
+	}))
+	s2.Object("Abilities", Abilities{}, schemabuilder.FetchObjectFromKeys(func(args struct{ Keys []*Abilities }) []*Abilities {
+		return args.Keys
+	}))
+
 	userWithContactInfo.Key("id")
 	userWithContactInfo.FieldFunc("secret", func(ctx context.Context, user *UserWithContactInfo) (string, error) {
 		return "shhhhh", nil
 	})
 
+	userWithContactInfo.FieldFunc("getSkills", func(ctx context.Context, user *UserWithContactInfo) (Skills, error) {
```

Comment:
nit: name the fieldfunc with noun, maybe just "skills".

Reply:
done